pkg/orchestration/wiring/ec2compute/common/common.go

```diff
@@ -139,11 +141,15 @@ func WireUp(microServiceNameInSpec, ec2ComputePlanName string, stateResource *or
 
 	for _, dependency := range dependencies {
 		if !dependency.Contract.IsEmpty() {
-			if dependency.Type != ups.ResourceType {
+			if dependency.Type != ups.ResourceType && dependency.Type != aws.DynamoDB {
```

Comment:
The idea of shapes is that you don't need to look at resource types.

Reply:
Agreed. This is unnecessary check. I'll remove it.